extras/parse_travis_log.py

```diff
@@ -177,6 +180,35 @@ def process_static_analysis(f, line):
         if line.startswith(' - clang-format for '):
             d.update(process_clang_format(f, filename))
 
+def process_pep8(f, line):
+    """
+    Process PEP8 output for a certain file.
+    """
+    # Check PEP8 on file pynest/nest/lib/hl_api_connections.py:
+    filename = line.split(' ')[-1].strip()[0:-1]
+    d = set()
+    res = {filename: d}
+    while True:
```

Comment:
@tammoippen 

Sounds reasonable. I see in the documentation that the read-ahead buffer is broken, that it's only used in next calls but not readline calls.  It's even built on top of a FILE*, so that you have buffers of buffers on top of the POSIX buffers on top of the hardware buffers...
(This is also the answer to why I'm suspicious of all directives coming out of that design camp)